Microsoft Academic.js

```diff
@@ -147,70 +120,70 @@ function doWeb(doc, url) {
 //Scrape a list of urls by extracting the pubID in each url, call the
 //API to receive the data and create an item in Zotero out of this.
 function scrape(urlList) {
-	for (let index in urlList) {
-		var url = urlList[index];
-		var pubID = url.match(/\/detail\/(\d+)/)[1];
-		
-		var apiUrl = "https://academic.microsoft.com/api/browse/GetEntityDetails?entityId=" +
-			pubID + "&correlationId=undefined";
+	for (let url of urlList) {
+		var pubID = url.match(/\/(?:detail|paper)\/(\d+)/)[1];
+		var apiUrl = "https://academic.microsoft.com/api/entity/" + pubID + "?entityType=2";
 		
 		ZU.doGet(apiUrl, function(text) {
-			//Z.debug(text)
 			var data = JSON.parse(text);
 			var type;
-			if (data.entity.c) {
-				type = "conferencePaper";
-			} else if (data.entity.j) {
-				type = "journalArticle";
-			} else {
-				type = "book";
+			switch (data.entity.v.entityType) {
+				case 0:
+					type = "book";
+					break;
+				// TODO what is case 2? find an example for it
```

Comment:
I tried to look a bit and looked at examples for patents, book chapters, others, and repository papers, none of which was 2 (they were either 0 or 3)